HelpMyStreetFE/HelpMyStreetFE/src/sass/components/profile/_job.scss

```diff
@@ -12,6 +12,62 @@
   }
 }
 
+@keyframes boing {
+    0% {
+        transform: scale(0.8);
+    }
+
+    80% {
+        transform: scale(1.01);
+    }
+
+    100% {
+        transform: scale(1.0);
+    }
+}
+
+.email-notification {
+
+
+    &-failure {
+        border-radius: 3px;
+        opacity: 0.9;
+        position: fixed;
+        right: 10px;
+        bottom: 10px;
+        padding-left: 10px;
+        padding-right: 10px;
+        visibility: hidden;
+        background-color: #B53737;
+
+        &.visible {
+            visibility: visible;
+            animation-name: boing;
+            animation-duration: 0.6s;
+            z-index: 5000;
+        }
+    }
+
+    &-success {
+        border-radius: 3px;
```

Comment:
Quite a lot of duplication here.  Could this styling be against one class rather than two?